Migration.md

```diff
@@ -12,9 +12,6 @@ release will remove the deprecated code.
 1. The macro `GZ_RENDERING_RESOURCE_PATH` is removed. Use
 `gz::rendering::getResourcePath()` instead.
 
-1. The macro `GZ_RENDERING_ENGINE_INSTALL_DIR` is removed. Use
-`gz::rendering::getEngineInstallDir()` instead.
```

Comment:
So a little confusing but there are 3 forms of `GZ_RENDERING_ENGINE_INSTALL_DIR`
* cmake variable in CMakeLists.txt
* C++ macro in config.hh.in
* environment variable

We removed the C++ macro (and kept the other two). So we can keep this entry in Migration.md

Reply:
That's right, thanks. [c383af1](https://github.com/gazebosim/gz-rendering/pull/1091/commits/c383af1a7eed3a95b03613c7221d3358688f9858)